src/handler.rs

```diff
@@ -272,16 +272,16 @@ mod tests {
 
         let resp: VerifyAttestationResponse =
             test::try_call_and_read_body_json(&app, req).await.unwrap();
-
-        assert_eq!(resp.signature, "61da9eef0a4597b674056e79c4e61f30cfb8f145d3355801a9def0d93a65b7295b8f64255cc7fac7b590a950a979fae3a274d91739f2d2c9284d518a2b9434511b");
-        assert_eq!(resp.secp256k1_public, "89b14cb02441b6850534580800bd0a33e6ca483a9ea8f0f55de0a99fbf4a4f02a525d6bb48a7a7a80928af68e0d4ad859d699b49538a425cd35403cd1fbdf956");
-        assert_eq!(resp.pcr0, "e32fe88a2ba4e70e2dd61decfc2063671eb4f26c0f68c00e6764bf47ecfd68ae98de726d1f814c9ff05cb3b17f3f0627");
+  
+        assert_eq!(resp.signature, "533b82a5c6e0bd4306a9852ffeff05904684d42f902b15888802371588a2c1832607a0f3b4b4331aa9fb278ebab02cd040e33e72922860dab5cf4920bc6c5e491c");
+        assert_eq!(resp.secp256k1_public, "d6e5fcbea3aa507a71c36ea8e3ac12db058653b054803a9ccb4f9f38dec704b5818e41e68f2f46bba87ab9ae88c7d853495c3427907c18c52f669c52697da391");
+        assert_eq!(resp.pcr0, "bee9db5a448337748ac99fdaa3289ff180c6f6bc7873703d882a4e3e22b1a671a2544751336a7a2e2d84a8776b12404c");
         assert_eq!(resp.pcr1, "bcdf05fefccaa8e55bf2c8d6dee9e79bbff31e34bf28a99aa19e6b29c37ee80b214a414b7607236edf26fcb78654e63f");
-        assert_eq!(resp.pcr2, "17ccf517a89089dd9fbfe48d1c5fc0db83008ac3008f41ecd4ea39f2a1df329ba122d892c00efb063846ba197ba7a1cc");
+        assert_eq!(resp.pcr2, "b9f3530d519e97e044becc80d98da4f2cd4adb3d0f67c33df5c854788431bf61db7fcef84046ac0c3087e945ff2cfbed");
         assert_eq!(
             resp.verifier_secp256k1_public,
             hex::encode(secp256k1_public)
         );
-        assert_eq!(resp.timestamp, 1711293221224);
+        assert_eq!(resp.timestamp, 1718876266772);
```

Comment:
try running test again, before the push.

Reply:
This data is failing now because the timestamp has gotten older than the validity period of an aws nsm attestation. I will update the tests with a fresh attestation doc but then again they will start falling after a period of time. To tackle this will need to fetch fresh attestation from a non-debug running nitro enclave which will create an external dependency and defeats the purpose of an unit test.

Reply:
Will add steps in the Readme for any developer trying to run the tests.

Reply:
Add steps in test file too or better if the  error returned can include this warning message to use latest attestation.